openassessment/templates/openassessmentblock/edit/oa_edit.html

```diff
@@ -99,15 +99,16 @@ <h2 class="openassessment_alert_title">{% trans "Rubric Change Impacts Settings
                     </div>
                     <p class="setting-help">{% trans "The date and time when students can no longer submit responses." %}</p>
                 </li>
-                <li id="openassessment_submission_image_wrapper" class="field comp-setting-entry">
+                <li id="openassessment_submission_file_wrapper" class="field comp-setting-entry">
                     <div class="wrapper-comp-setting">
-                        <label for="openassessment_submission_image_editor" class="setting-label">{% trans "Allow Image Responses"%}</label>
-                        <select id="openassessment_submission_image_editor" class="input setting-input" name="image submission">
-                            <option value="0">{% trans "False"%}</option>
-                            <option value="1" {% if allow_file_upload %} selected="true" {% endif %}>{% trans "True"%}</option>
+                        <label for="openassessment_submission_upload_selector" class="setting-label">{% trans "Allow File Upload"%}</label>
+                        <select id="openassessment_submission_upload_selector" class="input setting-input" name="upload submission">
+                            <option value="none">{% trans "None"%}</option>
```

Comment:
Can we make this clearer? Maybe, "No File Uploads Allowed" or something? Otherwise, will need to provide translators comments here.